### PR TITLE
kratos: add only i18n strings

### PR DESCRIPTION
**Pull request by elbotho**
this way we can start in crowdin without changing anything else.

### CODE REVIEW

File: src/data/en/index.ts [hunk 1]
```diff
@@ -943,3 +999,78 @@ export const loggedInData = {
...
+export const kratosMailStrings = {
+  recovery: {
+    valid: {
+      subject: '👉 Access to your Serlo account',
+      'body.plaintext': `👋 Hi {{ .Identity.traits.username }},
+are you trying to get access to your account again? If not please just ignore this mail.
+ 
+To reset your password please open the following link in your browser:
+{{ .RecoveryURL }}`,
+      body: `<p>👋 Hi <b>{{ .Identity.traits.username }}</b></p>,
```
Comment (hugotiburtino) [inline src/data/en/index.ts]:
```suggestion
      body: `<p>👋 Hi <b>{{ .Identity.traits.username }}</b>,</p>
```

File: src/data/en/index.ts [hunk 2]
```diff
@@ -943,3 +999,78 @@ export const loggedInData = {
     }
   },
 }
+
+export const kratosMailStrings = {
+  recovery: {
+    valid: {
+      subject: '👉 Access to your Serlo account',
+      'body.plaintext': `👋 Hi {{ .Identity.traits.username }},
+are you trying to get access to your account again? If not please just ignore this mail.
```
Comment (hugotiburtino) [inline src/data/en/index.ts]:
```suggestion
Are you trying to get access to your account at serlo.org? If not please just ignore this mail.
```

File: src/data/en/index.ts [hunk 3]
```diff
@@ -943,3 +999,78 @@ export const loggedInData = {
...
+ 
+To reset your password please open the following link in your browser:
+{{ .RecoveryURL }}`,
+      body: `<p>👋 Hi <b>{{ .Identity.traits.username }}</b></p>,
+<p>are you trying to get access to your account again? If not please just ignore this mail.</p>
```
Comment (hugotiburtino) [inline src/data/en/index.ts]:
```suggestion
<p>Are you trying to get access to your account at serlo.org? If not please just ignore this mail.</p>
```

**Review by hugotiburtino — CHANGES_REQUESTED** [review]

File: src/data/en/index.ts [hunk 4]
```diff
@@ -332,9 +333,64 @@ export const instanceData = {
...
+    auth: {
       pleaseLogInLink: 'Please log in',
       pleaseLogInText: 'to use this feature.',
+      registerTitle: 'Register your Serlo Account',
+      recoverTitle: 'Recover your account',
+      recoveryInstructions: 'Insert and submit your email address. We will then send you an email with a reset link.',
+      verify: {
+        title: 'Verify your email',
+        instructions: 'Insert and submit your email address to verify it.',
+        alreadyDone: 'Your are logged in, so you are very likely verified already.'
```
Comment (hugotiburtino) [inline src/data/en/index.ts]:
```suggestion
        alreadyDone: 'You are logged in, so you have already verified your email😊.'
```

File: src/data/en/index.ts [hunk 5]
```diff
@@ -332,9 +333,64 @@ export const instanceData = {
...
+        1040003: 'Continue',
+        1050001: 'Your changes have been saved! 🎉',
+        1060001: 'You successfully recovered your account. Please change your password in the next minutes.',
+        1060002: 'An email containing a recovery link has been sent to the email address you provided.',
+        1080001: 'An email containing a verification link has been sent to the email address you provided.',
+        1080002: 'You have successfully verified your email address.',
+        4000001: '%reason%',
+        4000002: '%field% is missing.',
+        4000003: 'Length must be \u003e= %expected_length%, but got %actual_length%.',
+        4000005: 'The password can not be used because %reason%.',
```
Comment (hugotiburtino) [inline src/data/en/index.ts]: Due to https://docs.google.com/document/d/1XtbBHgp9zarJUBQHaSA_tXtwtebaxVCFMJqeVZbVDb0/edit?disco=AAAAklr2SsA shouldn't it be?
```suggestion
        4000005: '%reason%',
```

**Review by hugotiburtino — CHANGES_REQUESTED** [review]

File: src/data/en/index.ts (hunk 5 above)
Comment (elbotho) [reply src/data/en/index.ts]: 👍 I will commit it like this for now.
the real problem here is that I don't know where the `%reason%`-string comes from. I'll investigate this.

File: src/data/en/index.ts [hunk 6]
```diff
@@ -943,3 +999,78 @@ export const loggedInData = {
     }
   },
 }
+
+export const kratosMailStrings = {
+  recovery: {
+    valid: {
+      subject: '👉 Access to your Serlo account',
+      'body.plaintext': `👋 Hi {{ .Identity.traits.username }},
+are you trying to get access to your account at serlo.org? If not please just ignore this mail.
```
Comment (hugotiburtino) [inline src/data/en/index.ts:1008]:
```suggestion
Are you trying to get access to your account at serlo.org? If not please just ignore this mail.
```
In English you should capitalize a word after greetings.

File: src/data/en/index.ts [hunk 7]
```diff
@@ -943,3 +999,78 @@ export const loggedInData = {
...
+ 
+To reset your password please open the following link in your browser:
+{{ .RecoveryURL }}`,
+      body: `<p>👋 Hi <b>{{ .Identity.traits.username }}</b>,</p>
+<p>are you trying to get access to your account at serlo.org? If not please just ignore this mail.</p>
```
Comment (hugotiburtino) [inline src/data/en/index.ts:1013]:
```suggestion
<p>Are you trying to get access to your account at serlo.org? If not please just ignore this mail.</p>
```

**Review by hugotiburtino — CHANGES_REQUESTED** [review]

File: src/data/en/index.ts (hunk 6 above)
Comment (elbotho) [reply src/data/en/index.ts:1008]: 👍

**Pull request merged**